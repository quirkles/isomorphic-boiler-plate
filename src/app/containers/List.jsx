import React from 'react';

const List = () => <div>Listy list</div>;

export default List;
