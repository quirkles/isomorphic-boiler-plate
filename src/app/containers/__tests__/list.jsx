import React from 'react';
import { mount } from 'enzyme';

import List from '../List';

describe('List component', () => {
  it('Renders without errors and matches snapshot', () => {
    let wrapper;
    expect(() => {
      wrapper = mount(<List />);
    }).not.toThrow();
    expect(wrapper).toMatchSnapshot();
  });
});
