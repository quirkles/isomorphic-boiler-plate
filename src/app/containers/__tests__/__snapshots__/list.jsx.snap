// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component Renders without errors and matches snapshot 1`] = `
<List>
  <div>
    Listy list
  </div>
</List>
`;
